lib/normalize.js

```diff
@@ -89,3 +87,50 @@ const normalizeAll = (dp) => {
 }
 module.exports.normalizeAll = normalizeAll
 
+const normalize = (path) => {
+  const writeDatapackage = (dp) => {
+    fs.writeFile(path, JSON.stringify(dp, null, 2),  (err) => {
+      if (err) {
+          console.error(err.message);
+          return;
+      };
+      console.log("Datapackage.json has been normalized");
+    });
+  }
+  
+  const readDatapackage = (path) => {
+    try {
+      return JSON.parse(fs.readFileSync(path, 'utf8'));
+    }
+    catch(err) {
+      console.error(err.message);
+      return;
+    }
+  }  
+  if(!path){
+    path = './datapackage.json'
+    let dp = readDatapackage(path)
+    normalizeAll(dp)
+    writeDatapackage(dp)
+  }
+  else if(fs.lstatSync(path).isFile()) {
+    let dp = readDatapackage(path)
+    normalizeAll(dp)
+    writeDatapackage(dp)
+  }
+  else if(path.slice(0,-1) != '/'){
```

Comment:
think we could avoid this else  if statement by simple path library
```
const  { join } = require('path')

// and down there it will automatically detect if there is no / and add
... else {
    path = join(path,  'datapackage.json')
    let dp = readDatapackage(path)
    normalizeAll(dp)
    writeDatapackage(dp)
 } 


```